DSharpPlus.Commands/Processors/TextCommands/TextLogging.cs

```diff
@@ -0,0 +1,10 @@
+namespace DSharpPlus.Commands.Processors.TextCommands;
+
+using System;
+using Microsoft.Extensions.Logging;
+
+internal static class TextLogging
+{
+    // Startup logs
+    public static readonly Action<ILogger, Exception?> MissingMessageContentIntent = LoggerMessage.Define(LogLevel.Critical, new EventId(1, "Text Commands Startup"), "TextCommandProcessor requires the MessageContents intent to be enabled. Please enable this intent in your DiscordClientConfiguration.");
```

Comment:
it doesn't. @-mentions to the bot still work without the intent, and DMs also work without the intent.

Reply:
I forgot about that.